src/types/Particle.ts

```diff
@@ -77,3 +77,50 @@ export const FLUKA_PARTICLE_TYPES = [
 		name: 'Electron'
 	}
 ] as const satisfies readonly ParticleType[];
+
+export const GEANT4_PARTICLE_TYPES = [
```

Comment:
That needs some comment here

- SHIELD-HIT12A supports COMMON_PARTICLE_TYPES 
- FLUKA supports COMMON_PARTICLE_TYPES + FLUKA_PARTICLE_TYPES 
- GEANT4 supports GEANT4_PARTICLE_TYPES 

I think this class needs some refactoring later.
As for now we can mark it down by some comment in the code and issue